packages/blade/src/components/Button/BaseButton/__tests__/BaseButton.web.test.tsx

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import renderWithTheme from '../../../../_helpers/testing/renderWithTheme.web';
+import BaseButton from '../BaseButton';
+
+beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
+afterAll(() => jest.restoreAllMocks());
+
+describe('<BaseButton />', () => {
+  it('should render button with default properties', () => {
```

Comment:
add test for icon-only button? and then you can do a negative assertion for a11y things below

Reply:
I've added tests for icon only button in later PRs. Specifically this one -> #484

Reply:
> and then you can do a negative assertion for a11y things below

Not sure what you mean by this?